### ISSUE TITLE
frequent crashes

### ISSUE THREAD

**Issue by zonderr** [CloudCompare/CloudCompare#1823]
My CloudCompare v2.12.4 flatpak crashes frequently on my Debian 5.5.17-1~bpo10+1 (2020-04-23) x86_64 GNU/Linux.

For example, if I load Sanchez_Guy_Expedition_cloud_cropped.laz and try to run RANSAC while unchecking cylinder and sphere, it crashes.

I was  not able to install another version (flatpak or not).

[Sanchez_Guy_Expedition_cloud_cropped.zip](https://github.com/CloudCompare/CloudCompare/files/12012392/Sanchez_Guy_Expedition_cloud_cropped.zip)

**Comment by rjanvier** [comment]
You are right. I can't reproduce it on Windows. Remembering a bit the code structure, It could be a memory alignement issue.
Anyway on my side, the plan is to remove Efficient RANSAC implementation from the flatpak because its license is not compatible with the GPL and restrictive ("for research purpose only")

**Comment by zonderr** [comment]
Too bad, I had research plans for it
